src/DIRAC/WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -101,194 +101,19 @@ def export_setJobStatus(cls, jobID, status="", minorStatus="", source="Unknown",
         Sets optionally the status date and source component which sends the status information.
         The "force" flag will override the WMS state machine decision.
         """
-        return cls.__setJobStatus(
-            int(jobID), status=status, minorStatus=minorStatus, source=source, datetime=datetime, force=force
+        return JobStatusUtility(cls.jobDB, cls.jobLoggingDB, cls.elasticJobParametersDB).setJobStatus(
+            int(jobID), status=status, minorStatus=minorStatus, source=source, force=force
```

Comment:
```suggestion
            int(jobID), status=status, minorStatus=minorStatus, source=source, dateTime=datetime, force=force
```
At least JobStatusUtility.setJobStatus still has a dateTime argument